c2corg_ui/templates/outing/view.html

```diff
@@ -50,6 +50,9 @@
   });
 
   module.value('documentData', {
+    "document_id": ${outing.get('document_id')},
+    "lang": ${dumps(lang) | n},
```

Comment:
On the other hand, "lang" is always a 2-letter code. Do we really need to
escape it?

2016-06-30 11:10 GMT+02:00 Arnaud Morvan <notifications@github.com>:

> In c2corg_ui/templates/outing/view.html
> <https://github.com/c2corg/v6_ui/pull/455#discussion_r69098659>:
>
> > @@ -50,6 +50,9 @@
> >    });
> >
> >    module.value('documentData', {
> > +    "document_id": ${outing.get('document_id')},
> > +    "lang": ${dumps(lang) | n},
>
> Yes lang is a string and it should be escaped using dumps, not
> html_entities which is the default from the template engine.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/c2corg/v6_ui/pull/455/files/99b2b6afcf7f784e22c46f66eb945aa9c2df7407#r69098659>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABIxi1K6g4r5v_gnKQglOjEhr4UkbyBUks5qQ4f7gaJpZM4I95MH>
> .
>